### DISCUSSION TITLE
How to register for github education pack .

### DISCUSSION THREAD

**Discussion by Dev-homestar** [community/community#137543] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi there,
how do i register for github education pack if university did not provide any educational email for students.

**Comment by EzraAriwomoi** [comment]
Take a look at [this](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/) from GitHub Blog.

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/137544#discussioncomment-10549780) for reference to why this discussion was closed and locked. Thanks!
